docs/modules/ROOT/pages/development.adoc

```diff
@@ -2,6 +2,48 @@
 
 This page describes considerations when developing RSpec-specific cops. It is intended to be a complement to the general https://docs.rubocop.org/rubocop/development.html[RuboCop development documentation].
 
+== Create a new cop
+
+NOTE: Clone the repository and run `bundle install` if not done yet.
+The following rake task can only be run inside the rubocop project directory itself.
+
+Use the bundled rake task `new_cop` to generate a cop template:
+
+[source,sh]
+----
+$ bundle exec rake 'new_cop[RSpec/CopName]'
+[create] lib/rubocop/cop/rspec/cop_name.rb
+[create] spec/rubocop/cop/rspec/cop_name_spec.rb
+[modify] lib/rubocop/cop/rspec_cops.rb - `require_relative 'rspec/cop_name'` was injected.
+[modify] A configuration for the cop is added into config/default.yml.
+Do 4 steps:
+  1. Modify the description of RSpec/CopName in config/default.yml
+  2. Implement your new cop in the generated file!
+  3. Commit your new cop with a message such as
+     e.g. "Add new `RSpec/CopName` cop"
+  4. Run `bundle exec rake changelog:new` to generate a changelog entry
```

Comment:
This doesn’t apply

Reply:
Oh.... This is a copy and paste of the log output when the new_cop command is executed with rubocop-rspec 😅

Reply:
I updated it.